pkg/porter/explain_test.go

```diff
@@ -324,6 +352,45 @@ func TestExplain_generatePrintableBundleCreds(t *testing.T) {
 	assert.Equal(t, 0, len(pb.Actions))
 }
 
+func TestExplain_generatePrintableBundlePorterVersion(t *testing.T) {
+	bun := bundle.Bundle{
+		Definitions: definition.Definitions{
+			"string": &definition.Schema{
+				Type:    "string",
+				Default: "clippy",
+			},
+		},
+		Custom: map[string]interface{}{
+			"sh.porter": map[string]interface{}{
+				"manifest": "5040d45d0c44e7632563966c33f5e8980e83cfa7c0485f725b623b7604f072f0",
+				"version":  "v0.30.0",
+				"commit":   "3b7c85ba",
+			},
+		},
+	}
+
+	pb, err := generatePrintable(bun, "")
+	assert.NoError(t, err)
+
+	assert.Equal(t, "v0.30.0", pb.PorterVersion)
+}
+
+func TestExplain_generatePrintableBundlePorterVersionNonPorterBundle(t *testing.T) {
```

Comment:
@carolynvs and this is the new test for the non-Porter bundle with no `Custom` property.